Magic/1. The Praeter.md

```diff
@@ -0,0 +1,24 @@
+# 1. About Magic: the Praeter
+
+## The praeter
+The praeter is a part, a dimension of the universe, whose mechanics and relation to the physical universe are not well understood. His access allows trained casters to perform magic. They can either enter the praeter directly or use sources of power displayed around the world.
+
+People aren't born with some gift to perform magic. Although anyone could enter it with the help of a source of power, the contact with the praeter is very exhausting and will basically faint any uninitiated.
+
+## The direct way
+Accessing the praeter directly requires years and years of training and a lot of energy. Thereby practically nobody can perform magic this way. Adepts of this methode are extremely powerful.
+
+It is said that some of them could even use the praeter to keep their spirit immortal and even come back in the real world as a newborn. They are sometimes perceived as divine or even as gods by certain folks.
+
+## Sources of power
+The most common way to access the praeter is through sources of power. These act as strong bonds to the praeter dimension and allow the casters to enter it easier.
+
+The only presence of great sources of power can weaken the people nearby and direct contact with a great source of power will basically faint any uninitiated.
+
+Sources of power are very heterogeneous in size and power. There are the inert sources of power like magical places and gemstones, and there are the living sources of power like magical beasts and the elves.
+
+The inert sources of power fade away as they are used and may regenerate over time. 
+
+Living sources gets exhausted when used and need time to recover, exactly like the caster. However, if pushed to much, they can also faint and even die. Nevertheless, they can defend themselves from the caster in a sort of mental duel.
```

Comment:
I am not sure about this mental duel, I think they should be consummed like coal.